tests/compile/src/test/kotlin/software/amazon/smithy/kotlin/codegen/util/TestUtils.kt

```diff
@@ -16,7 +16,7 @@ import java.io.ByteArrayOutputStream
 import java.io.File
 import java.io.OutputStream
 
-private const val SmithyVersion = "1.0"
+private const val SmithyVersion = "2.0"
```

Comment:
**Question**: Where is this used?

Reply:
Good question, I believe it's used in the test utils somewhere as the model version.